modules/apps/object/object-test/src/testFunctional/tests/ObjectsPopulating.testcase

```diff
@@ -420,12 +420,70 @@ definition {
 	}
 
 	@description = "LPS-138495 - Verify it is possible to map a Picklist for a Select From List field and view its entries"
-	@ignore = "Test Stub"
 	@priority = "4"
 	test CanMapPicklistTypeAndViewEntries {
+		Picklist.addPicklistViaAPI(picklistName = "Custom Picklist");
+
+		for (var itemsName : list "1,2") {
+			Picklist.addPicklistItemViaAPI(
+				itemKey = "${itemsName}",
+				itemName = "Item Test ${itemsName}",
+				picklistName = "Custom Picklist");
+		}
+
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object",
+			objectName = "CustomObject",
+			pluralLabelName = "Custom Objects");
+
+		ObjectAdmin.openObjectAdmin();
+
+		ObjectPortlet.selectCustomObject(label = "Custom Object");
+
+		ObjectAdmin.goToFieldsTab();
+
+		ObjectAdmin.addObjectFieldViaUI(
+			fieldLabel = "Field Picklist",
+			fieldPicklist = "Custom Picklist",
+			fieldType = "Picklist");
```

Comment:
Hi @andreefarias 

You need to use the macro that add by API,  you can use this: `ObjectAdmin.addObjectFieldViaAPI`. Let me know in case this macro doesn't work.